builder/parser/line_parsers.go

```diff
@@ -195,6 +194,35 @@ func parseLabel(rest string) (*Node, map[string]bool, error) {
 	return parseNameVal(rest, "LABEL")
 }
 
+// parses a statement with multiple keywords definition and name-value
+// assignment, like `name1 name2 name3=value`.
+// Note that this is a stricter format than the old format of assignment, in a way
+// that it only allows assignment of the form `name=value`. And that a name
+// definition or assignment to an empty value is designated by `name`.
```

Comment:
yes, you are right. I forgot to update this comment. Following is the correct description:

```
// Note that this is a stricter format than the old format of assignment, in a way
// that it only allows assignment of the form `name3=value` above. And that a name
// definition is of the form `name1` and `name2` above.
```